app/Services/Servers/SuspensionService.php

```diff
@@ -27,7 +28,7 @@ public function __construct(
      *
      * @throws \Throwable
      */
-    public function toggle(Server $server, string $action = self::ACTION_SUSPEND): void
+    public function handle(Server $server, string $action = self::ACTION_SUSPEND): void
```

Comment:
I'd like to remove the default action, while we're in here. Also let's switch it from a string to an enum!

```suggestion
    public function handle(Server $server, SuspendAction $action): void
```